content/docs/state-and-lifecycle.md

```diff
@@ -55,11 +55,11 @@ function tick() {
 setInterval(tick, 1000);
 ```
 
-[**Try it on CodePen**](http://codepen.io/gaearon/pen/dpdoYR?editors=0010)
+[**Przetestuj kod na CodePen**](http://codepen.io/gaearon/pen/dpdoYR?editors=0010)
 
-However, it misses a crucial requirement: the fact that the `Clock` sets up a timer and updates the UI every second should be an implementation detail of the `Clock`.
+Brakuje jeszcze fragmentu, który spełniałby kluczowe założenie: inicjalizacja timera i aktualizowanie UI co sekundę powinny być zaimplementowane w komponencie `Clock`.
```

Comment:
Słowo `timer` nie powinno być przetłumaczone? stoper/minutnik/czasomierz

Reply:
W pierwszej wersji przetłumaczyłem jako "czasomierz", ale wyglądało to dziwnie. Poza tym, słowo "timer" przyjęło się do języka polskiego (https://sjp.pwn.pl/sjp/timer;2578187.html).